app/client/__tests__/components/payment/cardInputForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+// import { render, fireEvent, waitFor } from "@testing-library/react";
```

Comment:
probably don't need this line here ... it should be pretty clear we need to add waitFor back in when/if we get to uncomment out the stripe tests